cryptography/Cargo.toml

```diff
@@ -36,36 +36,6 @@ bench = false
 portable = ["blst/portable"]
 
 [[bench]]
-name="dkg_recovery"
+name="bls12381"
```

Comment:
this part gets passed as the `module` name when the benchmarks are running and `module_path!()` is used to print something like `bls12381::dkg_reshare_recovery`